snuba/datasets/transactions_processor.py

```diff
@@ -110,6 +110,9 @@ def _process_base_event_values(
         processed["transaction_name"] = _unicodify(
             event_dict["data"].get("transaction") or ""
         )
+        processed["transaction_source"] = _unicodify(
+            event_dict["data"].get("transaction_info", {}).get("source") or ""
```

Comment:
Just to be extra careful, could we write it like this?

```suggestion
            (event_dict["data"].get("transaction_info") or {}).get("source") or ""
```

That would also cover the case where the dict contains an explicit `None`.